### PR TITLE
[zh-cn] init translation of specification_tables

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

complete meta docs

### Related issues and pull requests

part of #11063

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
最后三处我给了“意译”的建议，感觉直译稍微有一些生硬

File: files/zh-cn/mdn/writing_guidelines/page_structures/specification_tables/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+`\{{Specifications}}` 宏根据页面元数据信息的值生成了规范表格。
+
+默认情况下，使用 `browser-compat` 键中的值。每个值都引用了 [browser-compat-data](https://github.com/mdn/browser-compat-data) 仓库中的一个特定功能及其相关的兼容性和规范信息。例如，{{cssxref("text-align")}} 页有以下键，它用来获取相关的规范信息。
+
+```md
+browser-compat: css.property.text-align
+```
+
+有些功能在不在上述仓库中维护。在这些情况下，可以使用 `spec-urls` 键将规范信息添加到页面元数据中。例如，[`aria-atomic`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-atomic) 属性拥有以下页面元数据：
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/mdn/writing_guidelines/page_structures/specification_tables/index.md:30]:
```suggestion
有些特性不在上述仓库中维护。在这些情况下，可以使用 `spec-urls` 键将规范信息添加到页面元数据中。例如，[`aria-atomic`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-atomic) 属性拥有以下页面元数据：
```

File: files/zh-cn/mdn/writing_guidelines/page_structures/specification_tables/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+{{Specifications}}
+
+## 非标准特性
+
+当撰写非标准特性（特别是一个不再被标准追踪特性）的文档时，不要调用 `\{{Specifications}}` 宏。
+
+相反，尽量提供有关标准化状况和可能的替代方案的信息。举例说明：
+
+- 这个方法已经不再由任何标准追踪。保留它是为了兼容的目的。请使用*这个其他方法*代替。
+- 这个方法最初属于 [DOM Level 2 Traversal and Range](https://www.w3.org/TR/DOM-Level-2-Traversal-Range/) 规范，但器不属于目前的 DOM 规范。此特性不再有望成为标准。
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/page_structures/specification_tables/index.md:49]: sorry for my typo 😢

```suggestion
- 这个方法最初属于 [DOM Level 2 Traversal and Range](https://www.w3.org/TR/DOM-Level-2-Traversal-Range/) 规范，但其不属于目前的 DOM 规范。此特性不再有望成为标准。
```

**Review by yin1999 — APPROVED** [review]
👍

**Pull request merged**